units/patch_intent_test.go

```diff
@@ -1167,6 +1167,7 @@ func (s *PatchIntentUnitsSuite) TestProcessMergeGroupIntent() {
 	tasks := []string{"bynntask"}
 	s.verifyPatchDoc(dbPatch, j.PatchID, baseSHA, false, variants, tasks)
 	s.projectExists(j.PatchID.Hex())
+	s.Equal(dbPatch.Branch, "main")
```

Comment:
nit: swap the parameters (expected value first, actual second).

Same with the others below.